src/App.js

```diff
@@ -41,7 +41,18 @@ function App({ data }) {
     setActivities(newActivities);
     saveToLocal("localActivities", newActivities);
   }
+  //------------Delete Button -----------------
 
+  function handleDeleteButton(id) {
+    const filteredActivites = activities.filter(
+      (activity) => activity.id !== id
+    );
+
+    setActivities(filteredActivites);
+    saveToLocal("localActivities", filteredActivites);
+  }
+
+  //------------------------------------------------
```

Comment:
same here

Reply:
And another line of premium code gone... 😉